docs/sidebars.js

```diff
@@ -1,308 +1,321 @@
-                        'graph-recipe',
-                        'spaces',
-                    ],
-                },
-                {
-                    type: 'category',
-                    label: 'Actions',
-                    items: [
-                        'actions',
-                        'responses',
-                        'custom-actions',
-                        'forms',
-                        'default-actions',
-                        'slot-validation-actions',
-                    ],
-                },
-                {
-                    type: 'category',
-                    label: 'Evaluation',
-                    items: ['markers'],
-                },
-                {
-                    type: 'category',
-                    label: 'Channel Connectors',
-                    items: [
-                        {
-                            type: 'category',
-                            label: 'Text & Chat',
-                            items: [
-                                'connectors/facebook-messenger',
-                                'connectors/slack',
-                                'connectors/telegram',
-                                'connectors/twilio',
-                                'connectors/hangouts',
-                                'connectors/microsoft-bot-framework',
-                                'connectors/cisco-webex-teams',
-                                'connectors/rocketchat',
-                                'connectors/mattermost',
-                            ],
-                        },
-                        {
-                            type: 'category',
-                            label: 'Voice',
-                            items: ['connectors/audiocodes-voiceai-connect'],
-                        },
-                        'connectors/custom-connectors',
-                    ],
-                },
-                {
-                    type: 'category',
-                    label: 'Architecture', // name still confusing with architecture page elsewhere
-                    items: [
-                        'arch-overview',
-                        'tracker-stores',
-                        'event-brokers',
-                        'model-storage',
-                        'lock-stores',
-                        'secrets-managers',
-                        'nlu-only',
-                        'nlg',
-                    ],
-                },
-            ],
+          type: "category",
+          label: "Architecture",
+          items: [
+            "production/arch-overview",
+            "production/tracker-stores",
+            "production/event-brokers",
+            "production/model-storage",
+            "production/lock-stores",
+            "production/secrets-managers",
+            "production/nlg",
+          ],
         },
+        // TODO: create-task change content to remove DM1 topics
+        "production/testing-your-assistant",
```

Comment:
yeah, if we keep e2e testing in Pro, that means we will have a Pro only page for e2e testing, and that will be the only kind of automated testing we have in DM2

Reply:
yep